WcaOnRails/spec/requests/oauth/oauth_spec.rb

```diff
@@ -5,6 +5,17 @@
 RSpec.describe "oauth api" do
   include Capybara::DSL
 
+  # Pretend we're running on HTTPS, so that we can use the test server for redirect_uri.
+  before :all do
```

Comment:
Nice workaround! I wonder if this would be cleaner with an `around` block.

```ruby
around(:each) do |example|
  old_host = Capybara.default_host
  Capybara.default_host = Capybara.default_host.gsub("http", "https")
  old_protocol = default_url_options[:protocol]
  default_url_options[:protocol] = "https"

  example.run

  Capybara.default_host = old_host
  default_url_options[:protocol] = old_protocol
end
```

On retrospect, I'm not sure that that's actually any better... Please go ahead with whatever you prefer.